script/github-actions/get-changed-apps.js

```diff
@@ -17,6 +17,8 @@ const getManifests = filePath => {
   const rootAppFolder = filePath.split('/')[2];
   const fullAppPath = path.join(root, './src/applications', rootAppFolder);
 
+  if (!fs.existsSync(fullAppPath)) return [];
```

Comment:
Small update to catch an edge case. When an app directory is deleted or renamed, the files from the directory will be seen as changed. This function will throw an error trying to parse a non-existent directory.